### PR TITLE
Removes usage warning and tweaks UI

### PR DESCRIPTION
**Pull request by kenoir**
Changes:
- Adds usage count to tab label
- Label is greyed and non clickable when no usages available

With usages:
usages

Without usages (tab is not clickable):
usage

### CODE REVIEW

**Comment by kenoir** [comment]
Pinging @alastairjardine.

**Comment by alastairjardine** [comment]
Might be worth making it clickable, and putting in an informative message when there is zero usage on the resulting tab view.

It probably needs something to explain why it's not clickable otherwise.

**Comment by theefer** [comment]
:+1: 

and +1 to what @alastairjardine said.

**Comment by kenoir** [comment]
@alastairjardine Previously the tab said "None" when it was clickable. 

My thinking was that there should be some obvious differentiation on the image page as to whether an image has usages or not - greying and disabling the usage tab was my solution.

**Comment by kenoir** [comment]
Updated wording when no usages are available. Tab still remains unavailable as I think we can convey "no usages" without having to force the user to: click and see usages empty and then click again to return to metadata view.

File: kahuna/public/js/image/view.html [hunk 1]
```diff
@@ -160,14 +160,21 @@
...
+                    'radio-list--disabled': ctrl.usagesCount < 1,
+                    'radio-list--selected': ! showMetadata
+                   }"
+                   ng:click="(ctrl.usagesCount < 1) || (showMetadata = false)"
                    gr:track-click="Image panel tab"
                    gr:track-click-data="{ tab: 'Usage' }">
                 <div class="radio-list__selection-state"></div>
-                <div class="radio-list__label-value">Usage</div>
+                <div class="radio-list__label-value">
+                    Usages ({{ctrl.usagesCount || "None"}})
```
Comment (theefer) [inline kahuna/public/js/image/view.html]: I personally favour [not using integers as booleans](https://github.com/guardian/flexible-content/wiki/JavaScript-Style-Guide#syntax) for clarity. Could use a ternary operator instead if you agree, but happy for others to chip in on this coding style.

**Comment by theefer** [comment]
Minor comment on coding style, else :+1:

File: kahuna/public/js/image/view.html (hunk 1 above)
Comment (kenoir) [inline kahuna/public/js/image/view.html]: A ternary is more descriptive and not significantly more code. Have replaced.

**Comment by akash1810** [comment]
> without having to force the user to: click and see usages empty and then click again to return to metadata view.

Could we not render the usage tab at all then when there's no usage?

**Comment by kenoir** [comment]
@akash1810 I think that might not send the signal strongly enough that there are no usages, it might just make folks wonder where it's gone.

**Pull request merged**